service/controller/v2/arm_templates/vmss_worker.json

```diff
@@ -0,0 +1,282 @@
+                            "ssh": {
+                                "publicKeys": [
+                                    {
+                                        "keyData": "[parameters('vmssSshPublicKey')]",
+                                        "path": "[concat('/home/', parameters('vmssSshUser'), '/.ssh/authorized_keys')]"
+                                    }
+                                ]
+                            }
+                        }
+                    },
+                    "storageProfile": {
+                        "imageReference": {
+                            "publisher": "[parameters('vmssOsImagePublisher')]",
+                            "offer": "[parameters('vmssOsImageOffer')]",
+                            "sku": "[parameters('vmssOsImageSKU')]",
+                            "version": "[parameters('vmssOsImageVersion')]"
+                        },
+                        "osDisk": {
+                            "caching": "ReadWrite",
+                            "createOption": "FromImage",
+                            "managedDisk": {
+                                "storageAccountType": "[variables('vmssStorageAccountType')]"
+                            }
+                        },
+                        "copy": [
+                            {
+                                "name": "dataDisks",
+                                "count": "[length(parameters('vmssVmDataDisks'))]",
+                                "input": {
+                                    "caching": "ReadWrite",
+                                    "createOption": "Empty",
+                                    "diskSizeGB": "[parameters('vmssVmDataDisks')[copyIndex('dataDisks')].sizeGB]",
+                                    "managedDisk": {
+                                        "storageAccountType": "[variables('vmssStorageAccountType')]"
+                                    },
+                                    "lun": "[copyIndex('dataDisks')]"
+                                }
+                            }
+                        ]
+                    },
+                    "networkProfile": {
+                        "networkInterfaceConfigurations": [
+                            {
+                                "name": "[concat(parameters('vmssName'), '-nic')]",
+                                "properties": {
+                                    "enableIPForwarding": true,
+                                    "primary": "true",
+                                    "ipConfigurations": [
+                                        {
+                                            "name": "[concat(parameters('vmssName'), '-ipconfig')]",
+                                            "properties": {
+                                                "subnet": {
+                                                    "id": "[parameters('vmssVnetSubnetId')]"
```

Comment:
Only removed `loadBalancerBackendAddressPools` from vmss_worker.json, except that just a copy of initial vmss.json

Reply:
What happens when you set it to empty string instead?

Reply:
There are also options to make it conditional, we can talk if necessary. I'm against having another "vmss" json.